packages/ui/src/layout/banners/Announcement.tsx

```diff
@@ -52,15 +53,20 @@ const Announcement = ({
   }
 
   function handleLink() {
-    router.push(announcement.link)
+    // redirect to home page if on docs subpage
+    if (isDocsSubPage) {
```

Comment:
For simplicity, can we remove this special edge case for docs and always call `window.location.assign()` (just like the `<a>` tag)?